job-server/src/test/resources/db/h2/migration/V0_7_0__init_tables.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE JARS (
```

Comment:
Hmm.... is this exactly the same as the one in `src/main/resources`?  Should we have a symlink instead?   Would be very easy to forget to update the tables.